routes/drinks.js

```diff
@@ -0,0 +1,276 @@
+import express from 'express';
+import { createCloudinaryFolder, setupImageFolders, uploadImageToFolder } from '../db_config/cloudinary_config.js';
+import pool from '../db_config/db_connection.js'
+import { config } from 'dotenv';
+config();
+
+import swaggerUi from 'swagger-ui-express';
+import YAML from 'yamljs';
+
+const router = express.Router();
+
+const swaggerDocument = YAML.load('./documentary/swagger-specs.yaml');
+
+// mounting the Swagger UI middleware:
+router.use('/api-docs', swaggerUi.serve);
+router.get('/api-docs', swaggerUi.setup(swaggerDocument));
+
+
+async function uploadDrinkImage(imagePath) {
+  await setupImageFolders(); // Ensure the 'drinks-images' folder exists
+  const uploadResult = await uploadImageToFolder(imagePath, 'drinks-images');
+  console.log('Uploaded image:', uploadResult.secure_url);
+}
+
+// Call the function to upload an image
+//uploadDrinkImage('./images/purple.png');
+
+
+// Get all recipes
+router.get('/', async (req, res) => {
+  try {
+    const query = 'SELECT * FROM Drinks';
+    const { rows } = await pool.query(query);
+    res.json(rows);
```

Comment:
what happens if the rows are empty?

Reply:
this issue has just been handled by providing a concise message to the user to indicate that no drinks are available.